### ISSUE TITLE
"Object reference not set to an instance of an object." when login triggered on existing session

### ISSUE THREAD

**Issue by onionhammer** [Azure-Samples/active-directory-b2c-advanced-policies#9]
If the user attempts to invoke the login after they already have an active session cookie in B2C using `AAD-UserReadUsingAlternativeSecurityId`, an object reference error occurs.

If a pre-requisite is missing, it should display a more helpful error

Still investigating...

**Comment by pratik28** [comment]
Hello, 
Did you find a solution to this problem ?
I suppose it could be a problem with Windows - https://support.microsoft.com/en-ca/help/2379592/-object-reference-not-set-to-an-instance-of-an-object-error-message-wh 

Thanks

**Comment by onionhammer** [comment]
@pratik28 Yes, MS support had me add in the following line to one of my TechnicalProfiles.

```xml
          <UseTechnicalProfileForSessionManagement ReferenceId="SM-Noop" />
```
